server/src/main/java/se/fortnox/reactivewizard/server/ReactorServerConfigurer.java

```diff
@@ -0,0 +1,7 @@
+package se.fortnox.reactivewizard.server;
+
+import reactor.netty.http.server.HttpServer;
+
+public interface ReactorServerConfigurer {
```

Comment:
Beware that ordering might be important for some configurations, as channel configurations in netty has a strict ordering.

Reply:
Added prio